PublicationsRouter/v2/api/API.md

```diff
@@ -31,6 +31,7 @@ This page has the following contents:
   
  
 * [API for retrieving notifications](./API.md#api-for-retrieving-notifications)
+* [API for changing matching configuration](./API.md#api-for-matching-configuration)
```

Comment:
1. The opening para will need changing, currently "Jisc Publications Router API (PubRouter API)
Jisc Publications Router exposes an REST API that may be used to submit and/or retrieve publication notifications. These pages document the API interfaces and data formats."

2. I'm thinking perhaps this page is just too long now, and we should split it into 3 separate pages, keep this page as the jumping off page (and make it an index.md as Jake suggested) - and then have separate pages for:
  * Inbound
  * Outbound
  * Config